5.14.0-alpine/Dockerfile

```diff
@@ -9,7 +9,7 @@ ENV ACTIVEMQ_HOME /opt/activemq
 RUN set -x && \
   mkdir -p /opt && \
   apk --update add --virtual build-dependencies curl && \
-  curl -s http://archive.apache.org/dist/activemq/$ACTIVEMQ_VERSION/$ACTIVEMQ-bin.tar.gz | tar xvz -C /opt && \
+  curl -s https://archive.apache.org/dist/activemq/$ACTIVEMQ_VERSION/$ACTIVEMQ-bin.tar.gz | tar xvz -C /opt && \
```

Comment:
We should do this for every Docker file in a separate pull request, created #11 to track this.